docs/insights-archive-sample/namespaces/openshift-logging/lokistack-sample.json

```diff
@@ -0,0 +1 @@
rsion\":\"637293\",\"uid\":\"f105635e-dc45-491c-b778-540990e04b3f\"},\"spec\":{\"hashRing\":{\"type\":\"memberlist\"},\"limits\":{\"global\":{\"queries\":{\"queryTimeout\":\"3m\"}},\"tenants\":{\"application\":{\"retention\":{\"days\":1,\"streams\":[{\"days\":4,\"priority\":1,\"selector\":\"{kubernetes_namespace_name=~\\\"test.+\\\"}\"},{\"days\":3,\"priority\":1,\"selector\":\"{log_type=\\\"infrastructure\\\"}\"}]}}}},\"managementState\":\"Managed\",\"size\":\"1x.small\",\"storage\":{\"schemas\":[{\"effectiveDate\":\"2020-10-11\",\"version\":\"v11\"}],\"secret\":{\"name\":\"builder-dockercfg-jzlkq\",\"type\":\"azure\"}},\"storageClassName\":\"gp2-csi\",\"tenants\":{\"mode\":\"openshift-logging\"}},\"status\":{\"components\":{\"compactor\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"distributor\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"gateway\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"indexGateway\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"ingester\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"querier\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"queryFrontend\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]},\"ruler\":{\"Failed\":[],\"Pending\":[],\"Ready\":[],\"Running\":[]}},\"conditions\":[{\"lastTransitionTime\":\"2024-11-14T08:52:30Z\",\"message\":\"Invalid object storage secret contents: missing secret field: environment\",\"reason\":\"InvalidObjectStorageSecret\",\"status\":\"True\",\"type\":\"Degraded\"}],\"storage\":{}}}\n","loki.grafana.com/rulesDiscoveredAt":"2024-11-14T00:05:44Z"},"creationTimestamp":"2024-11-18T12:10:40Z","generation":1,"name":"lokistack-sample","namespace":"openshift-logging","resourceVersion":"54569","uid":"5072dcdd-e906-4b5f-8473-3d4aa2e579ba"},"spec":{"hashRing":{"type":"memberlist"},"limits":{"global":{"queries":{"queryTimeout":"3m"}},"tenants":{"application":{"retention":{"days":1,"streams":[{"days":4,"priority":1},{"days":3,"priority":1}]}}}},"managementState":"Managed","size":"1x.small","storage":{"schemas":[{"effectiveDate":"2020-10-11","version":"v11"}],"secret":{"name":"builder-dockercfg-jzlkq","type":"azure"}},"storageClassName":"gp2-csi","tenants":{"mode":"openshift-logging"}},"status":{"components":{"compactor":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"distributor":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"gateway":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"indexGateway":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"ingester":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"querier":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"queryFrontend":{"Failed":[],"Pending":[],"Ready":[],"Running":[]},"ruler":{"Failed":[],"Pending":[],"Ready":[],"Running":[]}},"conditions":[{"lastTransitionTime":"2024-11-18T12:10:48Z","message":"Missing object storage secret","reason":"MissingObjectStorageSecret","status":"True","type":"Degraded"}],"storage":{}}}
```

Comment:
can you please make the JSON in pretty format?

Reply:
Already updated!